### ISSUE TITLE
[Bug] Switching from Testnet to Mainnet does not correct the wallet.dns_server 

### ISSUE THREAD

**Issue by jfawChia** [Chia-Network/chia-blockchain#11518]
### What happened?

STEPS TO REPRODUCE:

`.\chia configure -t t`
Verify in config.yaml file that dns_server entry under wallet is `dns-introducer-testnet10.chia.net`

`.\chia configure -t f`
Verify in config.yaml file that dns_server entry under wallet is still `dns-introducer-testnet10.chia.net`

The value of dns_servers does not get reverted back to `dns-introducer.chia.net`

### Version

1.3.5

### What platform are you using?

Windows

### What ui mode are you using?

CLI

### Relevant log output

```shell
wallet:
  automatically_add_unknown_cats: false
  connect_to_unknown_peers: true
  database_path: wallet/db/blockchain_wallet_v2_CHALLENGE_KEY.sqlite
  db_sync: auto
  dns_servers:
  - dns-introducer-testnet10.chia.net
```

**Comment by wallentx** [comment]
Verified this is also an issue on Linux:

```
⌜william@firespring:~/.chia/../config⌟ » chia version
1.3.5

⌜william@firespring:~/.chia/../config⌟ » yq '. | (.selected_network, .farmer.selected_network, .full_node.selected_network, .harvester.selected_network, .introducer.selected_network, .pool.selected_network, .seeder.selected_network, .timelord.selected_network, .ui.selected_network, .wallet.selected_network, .full_node.dns_servers, .wallet.dns_servers)' config.yaml
mainnet
mainnet
mainnet
mainnet
mainnet
mainnet
mainnet
mainnet
mainnet
mainnet
- dns-introducer.chia.net
- dns-introducer.chia.net

⌜william@firespring:~/.chia/../config⌟ » chia configure -t t
Setting Testnet
Default full node port, introducer and network setting updated
Restart any running chia services for changes to take effect

⌜william@firespring:~/.chia/../config⌟ » yq '. | (.selected_network, .farmer.selected_network, .full_node.selected_network, .harvester.selected_network, .introducer.selected_network, .pool.selected_network, .seeder.selected_network, .timelord.selected_network, .ui.selected_network, .wallet.selected_network, .full_node.dns_servers, .wallet.dns_servers)' config.yaml
testnet10
testnet10
testnet10
testnet10
testnet10
testnet10
testnet10
testnet10
testnet10
testnet10
- dns-introducer-testnet10.chia.net
- dns-introducer-testnet10.chia.net

⌜william@firespring:~/.chia/../config⌟ » chia configure -t f
Setting Mainnet
Default full node port, introducer and network setting updated
Restart any running chia services for changes to take effect

⌜william@firespring:~/.chia/../config⌟ » yq '. | (.selected_network, .farmer.selected_network, .full_node.selected_network, .harvester.selected_network, .introducer.selected_network, .pool.selected_network, .seeder.selected_network, .timelord.selected_network, .ui.selected_network, .wallet.selected_network, .full_node.dns_servers, .wallet.dns_servers)' config.yaml
mainnet
mainnet
mainnet
mainnet
mainnet
mainnet
mainnet
mainnet
mainnet
mainnet
- dns-introducer.chia.net
- dns-introducer-testnet10.chia.net```

**Comment by emlowe** [comment]
This was fixed in 1.4.0 - closing
